css-color-4/Overview.bs

```diff
@@ -129,7 +129,7 @@ Representing sRGB Colors: the <<color>> type</h2>
 	<dfn>&lt;color></dfn> = <<rgb()>> | <<rgba()>> | <<hsl()>> | <<hsla()>> |
 	          <<hwb()>> | <<gray()>> | <<device-cmyk()>> |
 	          <<hex-color>> | <<named-color>> | currentcolor |
-	          <<deprecated-system-color>>
+	          <<system-color>>
```

Comment:
It looks like you define both `<system-color>` and `<deprecated-system-color>`, so don't both of them still need to be referenced here?  Though I think it would likely be better to have a single term that includes both of them (probably `<system-color>`), but that doesn't look like how it's defined in this PR.